charts/kueue/README.md

```diff
@@ -0,0 +1,59 @@
+# Kueue's helm chart
```

Comment:
Should this go in the website instead?
https://github.com/kubernetes-sigs/kueue/tree/main/site/content/en/docs/installation

however, better do so in a separate PR targeting the `website-release-0.4` branch.

Reply:
Certainly. This task is intended for future release.